rest-api/offline/biobank_samples_pipeline.py

```diff
@@ -315,6 +326,48 @@ def _get_hpo_type_sql(hpo_alias):
   result += "ELSE 'UNKNOWN' END)"
   return result
 
+
+def _get_status_flag_sql():
+  result = """
+      CASE
+        WHEN biobank_order.order_status = {amended} THEN 'edited'
```

Comment:
optional: "edited" vs "amended" here (and throughout, e.g. column names)? Should it be "amended" for consistency?

Reply:
Actually the value "edited" and the column name is from the requirement document. That's why we have this unconsistency issue.